_posts/2017-10-18-obs-dashboard.markdown

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+img: obs-dashboard.png
+title: OBS team dashboard
```

Comment:
I wouldn't put this up here, it has no real value for anyone else...

Reply:
So, just team dashboard?

Reply:
No I mean the whole thing.

Reply:
why not? it is open source, we are labelling and documenting junior issues and we even have some people interesting and contributor already. The idea was to collect projects were new people could start. What is the problem if only we use it? :thinking:

Reply:
In my opinion this might take away contributors from our other mentoring projects that have and impact for the openSUSE community. The dashboard was also meant as some some team internal exercise...